module/documents/item.mjs

```diff
@@ -787,23 +787,26 @@ export default class Item5e extends Item {
     if ( resourceUpdates.length ) await this.actor.updateEmbeddedDocuments("Item", resourceUpdates);
 
     // Prepare card data & display it if options.createMessage is true
-    const cardData = item.displayCard(options);
+    const cardData = await item.displayCard(options);
 
     // Initiate measured template creation
+    let templates;
     if ( config.createMeasuredTemplate ) {
-      const template = dnd5e.canvas.AbilityTemplate.fromItem(item);
-      if ( template ) await template.drawPreview();
+      try {
+        templates = await (dnd5e.canvas.AbilityTemplate.fromItem(item))?.drawPreview();
+      } catch(err) {}
     }
 
     /**
      * A hook event that fires when an item is used, after the measured template has been created if one is needed.
      * @function dnd5e.useItem
      * @memberof hookEvents
-     * @param {Item5e} item                  Item being used.
-     * @param {ItemUseConfiguration} config  Configuration data for the roll.
-     * @param {ItemUseOptions} options       Additional options for configuring item usage.
+     * @param {Item5e} item                                Item being used.
+     * @param {ItemUseConfiguration} config                Configuration data for the roll.
+     * @param {ItemUseOptions} options                     Additional options for configuring item usage.
+     * @param {MeasuredTemplateDocument[]|null} templates  The measured templates if they were created.
      */
-    Hooks.callAll("dnd5e.useItem", item, config, options);
+    Hooks.callAll("dnd5e.useItem", item, config, options, templates ?? null);
```

Comment:
Do we/core explicitly return `null` in any other hooks or should we let this slip through as `undefined`?

Reply:
I think null is fine here.